### ISSUE TITLE
Design for the modal

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Authors#1051]
We've never done a modal like this before, but it's worth trying.

I wonder what a "PublishPress" design would look like here?

<img width="561" alt="Screen Shot 2022-11-10 at 11 55 19 AM" src="https://user-images.githubusercontent.com/3868028/201158306-8bd69c36-c1b9-4c4b-a8f8-f3060c494732.png">

**Comment by ojopaul** [comment]
@stevejburge Do you want me to check a specific modal?

I'm asking because you said "I wonder what a "PublishPress" design would look like here?"

More like you're referencing a specific modal or it's simply about adding a modal(i can add any modal)?

**Comment by stevejburge** [comment]
@ojopaul @htmgarcia Yes, can we do bit more? Even just make it purple and add a Pro button?

**Comment by ojopaul** [comment]
@htmgarcia Can you reference to where i can see this?

**Comment by htmgarcia** [comment]
> @htmgarcia Can you reference to where i can see this?

@ojopaul please check the Blocks plugin:
![Screen Shot 2022-11-16 at 10 10 36](https://user-images.githubusercontent.com/4999794/202233437-d5aa1371-7339-471a-9f85-95ddec33ec73.png)

**Comment by stevejburge** [comment]
There we go :)

Let's follow our normal color scheme which is yellow for "Buy" buttons. right?

**Comment by htmgarcia** [comment]
> Let's follow our normal color scheme which is yellow for "Buy" buttons. right?

@stevejburge yes

**Comment by htmgarcia** [comment]
The close button could be optional imho, due the "x" to close the popup is already on the top-right corder.

**Comment by stevejburge** [comment]
Agree
